pkgs/development/libraries/qt-5/5.15/default.nix

```diff
@@ -339,5 +339,6 @@ let
 
   finalScope = baseScope.overrideScope(final: prev: {
     qttranslations = bootstrapScope.qttranslations;
+    qutebrowser = final.callPackage ../../../../applications/networking/browsers/qutebrowser { };
```

Comment:
I don't think this is the right place, as this file is mainly about building Qt 5 itself, just like `qt5.*`. Could you please put the `callPackage` in

- `pkgs/top-level/qt5-packages.nix` so that the package is accessible with `libsForQt5.qutebrowser` or
- `all-packages.nix`, with package name `qutebrowser-qt5`?

Reply:
I don't think `qutebrowser-qt5` is the right name either.

This is the package that was called `qutebrowser` before that name was stolen from it.

Reply:
With qutebrowser 3.0 qt6 is recommended and used by default, It makes sense to use it as the default version in the main package, like many other distros do. Qt5 is specifically mentioned as a fallback in the release notes. Also, the qt6 version ships a much newer version of chromium 108 vs 87, thus a better experience by default, is there any specific reasons why you need qt5 qutebrowser at all?

Reply:
> is there any specific reasons why you need qt5 qutebrowser at all?

Qt 5 can be cross-compiled while Qt 6 cannot.

Reply:
> > is there any specific reasons why you need qt5 qutebrowser at all?
> 
> Qt 5 can be cross-compiled while Qt 6 cannot.

In that case adding qutebrowser-qt5 seems fine.